Sources/Commands/SwiftTestTool.swift

```diff
@@ -296,7 +296,7 @@ public struct SwiftTestTool: SwiftCommand {
                 swiftTool.executionStatus = .failure
             }
 
-            if self.options.enableCodeCoverage {
+            if self.options.enableCodeCoverage, ranSuccessfully {
```

Comment:
Can this behavior be tested to avoid regressions?

Reply:
Probably, but I don't see any existing testing of the code-coverage feature at all.